packages/aggregator/src/connections/EntityConnection.js

```diff
@@ -7,22 +7,12 @@ import { ApiConnection } from '@tupaia/server-boilerplate';
 
 const { ENTITY_SERVER_API_URL = 'http://localhost:8050/v1' } = process.env;
 
-const createDescendantsUrl = (hierarchyName, entityCodes, dataSourceEntityType) =>
-  `hierarchy/${hierarchyName}/descendants?entities=${entityCodes.join(
-    ',',
-  )}&field=code&descendant_filter=type:${dataSourceEntityType}`;
-
-const createRelationsUrl = (
-  hierarchyName,
-  entityCodes,
-  aggregationEntityType,
-  dataSourceEntityType,
-) =>
-  `hierarchy/${hierarchyName}/relations?entities=${entityCodes.join(
-    ',',
-  )}&field=code&groupBy=descendant&${
-    aggregationEntityType === 'requested' ? '' : `ancestor_filter=type:${aggregationEntityType}`
-  }&descendant_filter=type:${dataSourceEntityType}`;
+const queryParams = (entityCodes, dataSourceEntityType) => ({
+  entities: entityCodes.join(','),
+  descendant_filter: `type:${dataSourceEntityType}`,
+  field: 'code',
+  groupBy: 'descendant',
```

Comment:
`groupBy` will be also used in the `descendants` query, I guess that's fine?

Reply:
Ah good point! That is technically incorrect, but practically fine... I should probs remove it tho since it's a bit confusing